Analyzer/include/Config.h

```diff
@@ -231,6 +231,22 @@ class Config
             };
             registerModules(tr, std::move(modulesList));
         }
+        else if (analyzer=="AnalyzeLepTrigger") {
```

Comment:
For my own sake can you keep the coding style for a particular file consistent. Please move the open bracket to the next line like I did the other if statments.

Reply:
Yes, sir.